### PR TITLE
Documentation: Add documentation for `Jadwal.php` #2

### PR DESCRIPTION
**Pull request by illusi0nary**
Added the phpdoc + return types for the Jadwal.php

### CODE REVIEW

**Comment by chez14** [comment]
Hi @illusi0nary,

Thank you for contributing, but we need more context to the functions. Can you add more information to it?

There are some Bahasa terms you can actually find on the internet since it's quite common.

- Jadwal = Schedule
- UTS (abbr. Ujian Tengah Semester) = Midterm Exam
- UAS (abbr. Ujian Akhir Semester) = Final Exam

Also good guess on the return types.

File: src/Data/Jadwal.php [hunk 1]
```diff
@@ -170,8 +182,11 @@ public function getUASes($refetch = false)
     }
 
 
-
-    protected function fetch($endpoint)
+    /**
+     * @param $endpoint
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    protected function fetch($endpoint) : void
```
Comment (chez14) [inline src/Data/Jadwal.php:193]: This function actually returns mixed, not `void`.

Comment (chez14) [reply src/Data/Jadwal.php:193]: Oh lol, my bad, this was my 2019 code, where my standard still a garbage lol. You guessed it right. Sorry.

**Comment by illusi0nary** [comment]
I updated the PR regarding to your feedback.

**Comment by chez14** [comment]
Nicely done. Thanks!

Adding label as per https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Pull request merged**